README.md

```diff
@@ -2,100 +2,68 @@
   <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
 </p>
 
-# Create a JavaScript Action using TypeScript
+# ![BYU logo](https://www.hscripts.com/freeimages/logos/university-logos/byu/byu-logo-clipart-128.gif) github-action-tf-plan-comment
 
-Use this template to bootstrap the creation of a JavaScript action.:rocket:
+GitHub Action to make a comment on a pull request with the proposed updated terraform plan
 
-This template includes compilication support, tests, a validation workflow, publishing, and versioning guidance.  
+This action takes in a JSON representation of your terraform plan and creates a comment on the Pull Request (PR) with basic info about what the plan will create, update, replace, or delete.
 
-If you are new, there's also a simpler introduction.  See the [Hello World JavaScript Action](https://github.com/actions/hello-world-javascript-action)
+**Note:** this action does not run terraform plan for you, you must pass in the plan as an input.
 
-## Create an action from this template
-
-Click the `Use this Template` and provide the new repo details for your action
-
-## Code in Master
-
-Install the dependencies  
-```bash
-$ npm install
-```
-
-Build the typescript and package it for distribution
-```bash
-$ npm run build && npm run pack
-```
-
-Run the tests :heavy_check_mark:  
-```bash
-$ npm test
-
- PASS  ./index.test.js
-  ✓ throws invalid number (3ms)
-  ✓ wait 500 ms (504ms)
-  ✓ test runs (95ms)
-
-...
-```
-
-## Change action.yml
-
-The action.yml contains defines the inputs and output for your action.
-
-Update the action.yml with your name, description, inputs and outputs for your action.
-
-See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-github-actions)
-
-## Change the Code
-
-Most toolkit and CI/CD operations involve async operations so the action is run in an async function.
-
-```javascript
-import * as core from '@actions/core';
-...
-
-async function run() {
-  try { 
-      ...
-  } 
-  catch (error) {
-    core.setFailed(error.message);
-  }
-}
-
-run()
+## Usage
+```yaml
+on: push
```

Comment:
This doesn't work on `push`, does it? https://github.com/byu-oit/github-action-tf-plan-comment/blob/1657df091461f53a2f9afc1df69166441aafdc08/src/main.ts#L11-L15

Reply:
[.github/workflows/test.yml](https://github.com/byu-oit/github-action-tf-plan-comment/blob/1657df091461f53a2f9afc1df69166441aafdc08/.github/workflows/test.yml#L4-L7) is set to run on `push`, too. I'd comment up there, but GitHub won't let me because it's not part of the diff for this PR.

Reply:
Fixed the README, in the test.yml file I still want the build to run on a push to the master branch. So I don't know what to do there.

Reply:
TBH, the README was the bigger issue since that's what people will copy and paste.

Maybe either split up the workflows or use an `if` statement to make the `test` job only run on pull requests? 🤷‍♂️

Speaking of which, I'm torn on how the action "succeeds" if it's not a pull request. One one hand, it means you can just stick it in any random workflow. On the other hand, should you?

Reply:
Good point. I put an if statement on the `test` job for now.